x-pack/plugins/reporting/kibana.jsonc

```diff
@@ -22,7 +22,8 @@
       "taskManager",
       "screenshotMode",
       "share",
-      "features"
+      "features", 
+      "kibanaUtils"
```

Comment:
Is this needed? I don't see added imports for `@kbn/kibana-utils-plugin` in reporting